ui/components/app/gas-details-item/gas-details-item-title/gas-details-item-title.js

```diff
@@ -21,23 +21,23 @@ const GasDetailsItemTitle = () => {
       <InfoTooltip
         contentText={
           <>
-            <Typography variant={TypographyVariant.H7}>
+            <Text variant={TextVariant.bodySm} as="h6">
```

Comment:
Seeing as this text seems to be inside the tooltip could you take a screenshot hovering over the (i) info icon to show the tooltip content

Reply:
I have added the new screenshots, please review them.
Thanks!